projects/packages/stats/src/class-options.php

```diff
@@ -155,12 +155,15 @@ public static function upgrade_options( $options ) {
 	 */
 	protected static function get_defaults() {
 		return array(
-			'admin_bar'    => true,
-			'roles'        => array( 'administrator' ),
-			'count_roles'  => array(),
-			'do_not_track' => true, // @todo
-			'blog_id'      => Jetpack_Options::get_option( 'id' ),
-			'version'      => Main::STATS_VERSION,
+			'admin_bar'                => true,
+			'roles'                    => array( 'administrator' ),
+			'count_roles'              => array(),
+			'do_not_track'             => true, // @todo
+			'blog_id'                  => Jetpack_Options::get_option( 'id' ),
+			'version'                  => Main::STATS_VERSION,
+			'collapse_nudges'          => false,
+			'enable_calypso_stats'     => false,
```

Comment:
```suggestion
			'enable_calypso_stats'     => true,
```
Could we set the default to true here and address https://github.com/Automattic/jetpack/issues/28709?

Also, what are your thoughts on referencing Odyssey instead of Calypso in the option name (e.g., `enable_odyssey_stats`)? I understand this might not be realistic due to backward compatibility concerns.

Reply:
@jsnmoon Ya two good points. I was planning to address #28709 with `enable_odyssey_stats` default to `true` for the all-in roll out.  

> I understand this might not be realistic due to backward compatibility concerns.

I'm not too concerned about this one since we haven't launched yet 😄 What do you think?

Reply:
Let's rename it to `enable_odyssey_stats` then 🙂

Reply:
Renamed in 60b7e07e54